<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>779</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>90</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(64, 191, 155);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="first_column">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>251</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #54D3AF;
                                      border-radius: 15px;
                                       padding: 6px;
                                       background-color: #54D3AF;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="second_column">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>471</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono NL</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #54D3AF;
                                      border-radius: 15px;
                                       padding: 6px;
                                       background-color: #54D3AF;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="section_name">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>7</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Math TeX Gyre</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>100</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="CPU">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:CPU.png);

QPushButton {
                           border: 2px solid #8f8f91;
                            border-radius: 10px;
                            padding: 6px;
                        }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RAM">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(97, 53, 131);
font: bold;
border-image: url(:RAM.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GPU">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(97, 53, 131);
font: bold;
border-image: url(:GPU.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MONITOR">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(97, 53, 131);
font: bold;
border-image: url(:MONITOR.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BASEBOARD">
       <property name="styleSheet">
        <string notr="true">color: rgb(97, 53, 131);
font: bold;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
